### PR TITLE
Document UPD

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a documentation.

I stole a picture from you @mikaell - please approve if its ok! 😸

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: docs/user-guide/cases.md [hunk 1]
```diff
@@ -22,13 +22,29 @@ It can be visualized by clicking on the 'Visualize report' link on the left side
...
 
 Rank model version shows which rank model that has been used to rank the variants in the case
 
 The "Activity" feed is a reverse chronological list of events and comments related to the case. Examples of events include comments, status updates, assignments, Sanger sequencing orders etc.
 
+### Cytogenomics
+The case page contains three (admin configuration optional) cytogenomics applications: [Gens](https://github.com/Clinical-Genomics-Lund/gens), [Chromograph](https://github.com/mikaell/chromograph) and [vcf2cytosure](https://github.com/NBISweden/vcf2cytosure).
+Gens integration is available via the "View CN profile" button on the case page as well as the "Copy number profile" links under SV variant summary on each SV variant page.
+Chromograph integration can be found under the Images cards on the case page. Chromograph will display ideograms, genome coverage from [TIDDIT](https://github.com/SciLifeLab/TIDDIT) (orange graph), autozygosity from [rhocall][rhocall] (gray bars) and UPD calls from [UPD.py](https://github.com/bjhall/upd). UPD calls are colored deep red for maternal UPD candidate regions,
```
Comment (northwestwitch) [inline docs/user-guide/cases.md:41]: You could add these links to then end of the page, like it is done for rhocall if you want to.

**Review by northwestwitch — APPROVED** [review]
👍

File: docs/user-guide/cases.md (hunk 1 above)
Comment (dnil) [reply docs/user-guide/cases.md:41]: Yeah, I use the logic of direct link for one mention, reference for two. I'm more worried about the images - seems the current .io doesn't show the others from the same page?

Comment (northwestwitch) [reply docs/user-guide/cases.md:41]: Also images from other pages like [dashboard](https://clinical-genomics.github.io/scout/user-guide/dashboard/) are missing.

Comment (northwestwitch) [reply docs/user-guide/cases.md:41]: Checking the documentation looks like we should add each image of the page to the yaml index (mkdocs.yml), which we don't:
https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-images-and-media

Comment (dnil) [reply docs/user-guide/cases.md:41]: Thanks, that link would have saved me some googling had I seen it! 😅 Anyway, still needed to come to terms with the abs/relative URL issue between local testing and deployment. For local testing, all the `[](/img/sth.png)` worked fine, but they fail when the site url for the same is `io.github.com/scout/img/sth.png`.

Comment (northwestwitch) [reply docs/user-guide/cases.md:41]: Nice with the doc pages fixing!

**Pull request merged**